CRaC tests are failing

While working on #652, I realised that CRaC tests have never been executed in CI.

`package-docker-crac` fails when executing the checkpoint image:

```
...
[INFO] [INFO] Step 10/10 : ENTRYPOINT ["/home/app/checkpoint.sh"]
[INFO] [INFO] 
[INFO] [INFO]  ---> Running in f2757c12d0bd
[INFO] [INFO] Removing intermediate container f2757c12d0bd
[INFO] [INFO]  ---> a7eda21c1905
[INFO] [INFO] Successfully built a7eda21c1905
[INFO] [INFO] Successfully tagged package-docker-crac-crac-checkpoint:latest
[INFO] [INFO] CRaC Checkpoint image: package-docker-crac-crac-checkpoint
[INFO] [INFO] Container started: 8b51aa6a139afcb23673cbd7ea9283341d8972ab9295c4c22ab2d29f918148c2 8b51aa6a139afcb23673cbd7ea9283341d8972ab9295c4c22ab2d29f918148c2
[INFO] [INFO] Waiting 60 seconds for completion
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] BUILD FAILURE
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] Total time:  01:05 min
[INFO] [INFO] Finished at: 2023-03-10T14:27:27Z
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] 4 goals, 4 executed
[INFO] [INFO] 
[INFO] [INFO] Publishing build scan...
[INFO] [INFO] https://ge.micronaut.io/s/va44p43zdexak
[INFO] [INFO] 
[INFO] [ERROR] Failed to execute goal io.micronaut.maven:micronaut-maven-plugin:4.0.0-SNAPSHOT:docker-crac (default-docker-crac) on project package-docker-crac: Image package-docker-crac-crac-checkpoint exited with code 1 -> [Help 1]
```

--- a/micronaut-maven-integration-tests/src/it/dockerfile-docker-crac/pom.xml
+++ b/micronaut-maven-integration-tests/src/it/dockerfile-docker-crac/pom.xml
@@ -71,6 +71,11 @@
       <artifactId>micronaut-test-junit5</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.micronaut</groupId>
+      <artifactId>micronaut-jackson-databind</artifactId>
+      <scope>runtime</scope>
+    </dependency>
 
 
     <!-- For CRaC -->

--- a/micronaut-maven-integration-tests/src/it/package-docker-crac-aot/pom.xml
+++ b/micronaut-maven-integration-tests/src/it/package-docker-crac-aot/pom.xml
@@ -74,6 +74,11 @@
       <artifactId>micronaut-test-junit5</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.micronaut</groupId>
+      <artifactId>micronaut-jackson-databind</artifactId>
+      <scope>runtime</scope>
+    </dependency>
 
     <!-- For CRaC -->
     <dependency>

--- a/micronaut-maven-integration-tests/src/it/package-docker-crac/pom.xml
+++ b/micronaut-maven-integration-tests/src/it/package-docker-crac/pom.xml
@@ -52,6 +52,11 @@
       <scope>compile</scope>
     </dependency>
     <dependency>
+      <groupId>io.micronaut</groupId>
+      <artifactId>micronaut-jackson-databind</artifactId>
+      <scope>runtime</scope>
+    </dependency>
+    <dependency>
       <groupId>ch.qos.logback</groupId>
       <artifactId>logback-classic</artifactId>
       <scope>runtime</scope>

